docs/specification/2025-03-26/basic/authorization.mdx

```diff
@@ -321,67 +268,6 @@ Servers **MUST** return appropriate HTTP status codes for authorization errors:
 3. Token rotation **SHOULD** be implemented for enhanced security
 4. Token lifetimes **SHOULD** be limited based on security requirements
```

Comment:
Also applicable to when the MCP server has its own AS rather than integrate an established one.